src/interface_adapter/rooms/RoomsState.java

```diff
@@ -11,13 +11,20 @@ public class RoomsState {
   private List<Room> availableRooms;
   private List<Message> displayMessages;
   private String sendMessage;
+  private String email; // Other user email
+  private String createRoom; // Room to create
```

Comment:
The naming of this field confused me for a min, maybe we could change the comment to say "the value of the room creation text field .name" or something?

Reply:
Fixed